caffe2/core/operator.cc

```diff
@@ -56,6 +56,14 @@ OperatorBase::OperatorBase(const OperatorDef& operator_def, Workspace* ws)
   type_ = operator_def.type();
 }
 
+OperatorBase::OperatorBase(
+    const c10::FunctionSchema& fn_schema,
+    const std::vector<c10::IValue>& inputs,
```

Comment:
I think using move semantics on such small datatypes (IValue is 8 bytes) will lead to potential misuse.  https://godbolt.org/z/ufro_u no compiler warnings even with -pedantic, -Wall

I have to wrap function schema in a unique pointer now to avoid a bug in nvcc, so moves don't matter